nomad/grapl-core.nomad

```diff
@@ -923,20 +956,23 @@ job "grapl-core" {
       driver = "docker"
 
       config {
-        image = "${var.container_registry}grapl/graphql-endpoint:${var.graphql_endpoint_tag}"
+        image = "${var.container_registry}grapl/${var.container_repo}graphql-endpoint:${var.graphql_endpoint_tag}"
         ports = ["graphql-endpoint-port"]
       }
 
+      template {
+        data        = local.local_vars
+        destination = "graphql-endpoint.env"
+        env         = true
+      }
+
       env {
         DEPLOYMENT_NAME               = var.deployment_name
         RUST_LOG                      = var.rust_log
         AWS_REGION                    = var.aws_region
```

Comment:
Looks like the JavaScript SDK is one that requires `AWS_REGION`: https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/setting-region.html

FUN.

Reply:
Because this is the lone `AWS_REGION` usage (as opposed to `AWS_DEFAULT_REGION`), it would be worthwhile to add a comment to the effect of "No, this isn't a mistake, the JavaScript SDK only recognizes `AWS_REGION`, not `AWS_DEFAULT_REGION`"